android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -248,7 +248,11 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
      */
     private fun attachPlayer(view: RNPlayerView, playerId: NativeId?, playerConfig: ReadableMap?) {
```

Comment:
What's the case of playerId being null here? I think we should log a no-op error if  `args?.getString(1)` in receiveCommand returns null.

Reply:
An error is now logged: 2cb51f81323d791aa502b8b5d3a61ad9203605b4

Reply:
What would you think about being more explicit in the `receiveCommand` function and then having playerId not nullable here?

Something like :
```
Commands.ATTACH_PLAYER -> {
    val playerId = args?.getString(1) ?: error("cannot attach a player without a playerId")
    val playerConfig = args.getMap(2)
    attachPlayer(view, playerId, playerConfig)
}
```

I would do this for all cases and make sure that parameters that cannot be null have to be provided. I would suggest a follow up ticket if you agree.

Reply:
Done: 8773de7fd55d2b9cc5cb989d80ef831ad03ea60a